IoTuring/Entity/Deployments/Wifi/Wifi.py

```diff
@@ -34,57 +60,103 @@ def Initialize(self):
         self.getWirelessMethods = {
             OsD.WINDOWS: self.GetWirelessStrength_Windows,
             OsD.LINUX: self.GetWirelessStrength_Linux,
-            OsD.MACOS: "self.GetWirelessStrength_Mac" # TODO
+            OsD.MACOS: "self.GetWirelessStrength_Mac",
```

Comment:
I don't think these quotes are wanted :)

Reply:
There is alot of stuff in the code that is commented out for development. But thanks for checking.